### ISSUE TITLE
A11y_AzureMachineLearningExtension_Subscriptions_Keyboard: Tooltips present for workspaces/toolbar icons/resources etc are not available for keyboard users.

### ISSUE THREAD

**Issue by NiharikaTipirishetty** [microsoft/vscode-tools-for-ai#1264]
**Github Tags:**
#SH-AzureMachineLearningExtension-Win32-Aug2021;#A11yMAS;#A11yTCS;#WCAG2.1.1;#Keyboard;#Win32;#DesktopApp;#A11ySev2;#Development and Test Resources for VS and VSCode AI Tools;

### Environment Details:
Visual Studio Code

### Note:
This issue is observed for all the controls in Machine learning tree view.

### Repro Steps:

1. Hit the Azure Machine Learning Studio URL and sign in with appropriate credentials.
2. Tab till Notebooks in the left pane and press enter. Select any file from the pane, tab till "Edit in VS code "option in command bar and press enter.
3. A pop up will appears on the screen to open VS Code app, tab till 'Open' button and press enter
4. Tab till Azure in the left pane and press enter, tab till any subscription ex: ML.Net Model Builder Demo and press enter.
5. Tab till any workspace ex: Basic Workspace or any toolbar icon i.e., create dataset/create job buttons under the subscription and press enter.
6. Verify when keyboard focus lands on the controls tooltip is visible or not

### Actual Result:
Tooltips present for workspaces/toolbar icons/resources etc are not available for keyboard users.

### Expected Result:
Tooltips present for workspaces/toolbar icons/resources etc should be available for keyboard users as well. When keyboard focus lands on any control, tooltip info should be visible.

### User Impact:
Keyboard users will not be able to understand the purpose of a icon/control if tooltip is not visible

**Comment by tbombach** [comment]
This behavior was logged for VSCode core as an umbrella issue for all interactions in the main UI: https://github.com/microsoft/vscode/issues/52016

The conclusion was that users with screen readers will hear the description of UI elements, which conveys the info that users see visually in tooltips. So the only users who this affects are ones who use the keyboard without a screen reader and without a mouse. The linked bug concluded that it is not possible for VSCode to activate a system tooltip without using the mouse (therefore our extension cannot activate the tooltips either).

**Comment by tbombach** [comment]
Closing this issue because it's not possible for us to trigger these tooltips without support from VSCode (which has chosen not to implement this)
